cypress/integration/core_opensearch_dashboards/apps/query_enhancement/dataset_selector_spec.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  MiscUtils,
+  TestFixtureHandler,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import { CURRENT_TENANT } from '../../../../../utils/commands';
+import { clusterName, clusterConnection } from '../../../../../utils/constants';
+
+const miscUtils = new MiscUtils(cy);
+const testFixtureHandler = new TestFixtureHandler(cy, Cypress.env('openSearchUrl'));
+
+const indexSet = ['logstash-2015.09.22', 'logstash-2015.09.21', 'logstash-2015.09.20'];
+
+describe('dataset navigator', { scrollBehavior: false }, () => {
+  before(() => {
+    CURRENT_TENANT.newTenant = 'global';
+    cy.fleshTenantSettings();
+  });
+
+  describe('empty state', () => {
+    it('no index pattern', function () {
+      // Go to the Discover page
+      miscUtils.visitPage(
+        `app/data-explorer/discover#/?_g=(filters:!(),time:(from:'2015-09-19T13:31:44.000Z',to:'2015-09-24T01:31:44.000Z'))`
+      );
+
+      cy.waitForLoaderNewHeader();
+      cy.getElementByTestId('discoverNoIndexPatterns');
+    });
+  });
+
+  describe('select indices', () => {
+    before(() => {
+      testFixtureHandler.importJSONMapping(
+        'cypress/fixtures/dashboard/opensearch_dashboards/query_enhancement/mappings.json.txt'
```

Comment:
we could consider a utils function that does this a little bit easier like we know we want to store in fixtures so then we just need to have the plugin define it data it wants to use.

but i dont really see why each plugin needs to define its own fixtures.

if we add a new plugin then the plugin will think it needs to keep adding more sample data that is increasing the resources taken for local development when this data could have been re-used

Reply:
agree on getting a utils function so importing datas can be cleaner. 

Currently changing the fixture file structure so that dashboard and plugin fixtures are not seperated, instead each folder are the data name and contain one data file and one mapping file. 

Ex: 

/fixtures
  /discover
      /data.json
      /mappings.json
  /logstash
      /data.json
      /mappings.json
  /timestamp
      /data.json
      /mappings.json
.....